about.html

```diff
@@ -0,0 +1,138 @@
+      </div>
+      <ul class="conta">
+        <li><a href="#about">About</a></li>
+        <li><a href="index.html#program">Program</a></li>
+        <li><a href="https://www.buildingsmart.org/about/openbim/">Join</a></li>
+        <li><a href="index.html#sponsors">Sponsor</a></li>
+        <li><a href="https://www.buildingsmart.org/resources/latest-news/">News</a></li>
+        <li><a id="aButton" href="https://www.buildingsmart.org/chapter-directory/">BIM Campaing</a></li>
+      </ul>
+    </nav>
+  </div>
+
+  <nav id="hiddenNav">
+    <ul>
+      <li><a href="about.html" onclick="redirect()">About</a></li>
+      <li><a href="#program" onclick="redirect()">Program</a></li>
+      <li><a href="https://www.buildingsmart.org/about/openbim/" onclick="redirect()">Join</a></li>
+      <li><a href="#sponsors" onclick="redirect()">Sponsor</a></li>
+      <li><a href="https://www.buildingsmart.org/resources/latest-news/" onclick="redirect()">News</a></li>
+    </ul>
+    <div id="closebtn"><img id="closebtn_img" src="static/icon-cancel.svg" alt="close button icon"></div>
+  </nav>
+
+  <section id="main" class="container">
+    <nav>
+      <i id="listIcon" class="fa-2x fa-solid fa-bars"></i>
+    </nav>
+    <div class="hero">
+      <h4 class="lato">"Open BIM tools for everyone!"</h4>
+      <h1 class="coco">OPEN BIM GLOBAL DEVELOPERS SUMMIT 2022</h1>
+      <div class="break"></div>
+      <p id="heroDescription">The Open BIM Global Summit brings togueter the community of experts,
+        researchers, and practitioners who comprise the Open BIM affiliate
+        network in a different country every two years. This year, we hope to
+        expand our invitation list including organizations and individuals interested in applying open BIM tools in
+        an effort to make BIM models more accessible to everyone and not dependant on authoring BIM tools.</p>
+      <p id="contact">Please contact us per Email for any further questions about Open BIM Global Summit 2022!</p>
+      <p id="email"><span>openBIM2022@openbim.org</span></p>
+    </div>
+  </section>
+  <section id="about" class="container-fluid">
+    <h2 id="mobil">Open BIM Global<br>Developers Summit Logo</h2>
+    <h2 id="desktop">Open BIM Global Developers Summit Logo</h2>
+    <div class="break"></div>
+    <p id="logoTextMobil">The logo of Open BIM Global Developers Summit 2022 was decided throught the Logo
+      competition from october 8th to november 8th.
+    </p>
+    <p id="logoTextDesktop">The logo of Open BIM Global Developers Summit 2022 was decided throught the Logo
+      competition from october 8th to november 8th. After online public voting and Selection Committee's judging
+      process,
+      the logo from Juan Perez won the prize. After consultation with Mr. Agrawal we optimized the prize-winning-logo
+      for
+      this year's summit as following.
+    </p>
+    <div class="logo">
+      <img src="static/openbim.jpg" alt="">
```

Comment:
- [ ] Please, try to fix alt issues, as I mentioned before 🥇

Reply:
✔️ well resolved